assets/pages/new-contributions-landing/contributionsLandingReducer.js

```diff
@@ -148,11 +159,11 @@ function createFormReducer(countryGroupId: CountryGroupId) {
         return { ...state, paymentMethod: action.paymentMethod };
 
       case 'UPDATE_PAYMENT_READY':
-        return action.paymentHandlers
+        return action.thirdPartyPaymentLibraries
           ? {
             ...state,
             paymentReady: action.paymentReady,
-            paymentHandlers: { ...state.paymentHandlers, ...action.paymentHandlers },
+            thirdPartyPaymentLibraries: { ...state.thirdPartyPaymentLibraries, ...action.thirdPartyPaymentLibraries },
```

Comment:
Unfortunately that won't work. The spread syntax performs a shallow copy, so any element in `action.thirdPartyPaymentLibraries` will overwrite those in `state.thirdPartyPaymentLibraries`, whereas what you want is to merge them together.

Reply:
Good point! Have updated, thanks.